src/components/LHNOptionsList/OptionRowLHN.js

```diff
@@ -57,8 +57,11 @@ const OptionRowLHN = (props) => {
     const optionItem = SidebarUtils.getOptionData(props.reportID);
 
     React.useEffect(() => {
+        if (!optionItem || !optionItem.isPinned) {
+            return;
+        }
         ReportActionContextMenu.hideContextMenu(false);
-    }, [optionItem.isPinned]);
+    }, [optionItem]);
 
     if (!optionItem) {
         return null;
```

Comment:
I assume I'm missing something here, but why is this still necessary?

Reply:
I think this is for performance

https://github.com/Expensify/App/blob/96adb8e58e53fd76c7a316234dab322643dfd479/src/libs/SidebarUtils.js#L193-L202

Reply:
Sorry, I was still cleaning up. Should be ready for review now